contracts/Vault.vy

```diff
@@ -1127,19 +1127,15 @@ def creditAvailable(_strategy: address = msg.sender) -> uint256:
 @internal
 def _expectedReturn(_strategy: address) -> uint256:
     # See note on `expectedReturn()`.
-    strategy_lastReport: uint256 = self.strategies[_strategy].lastReport
-    strategy_totalGain: uint256 = self.strategies[_strategy].totalGain
-    strategy_activation: uint256 = self.strategies[_strategy].activation
-
-    blockDelta: uint256 = block.number - strategy_lastReport
-    if blockDelta > 0:
+    delta: uint256 = block.timestamp - self.strategies[_strategy].lastReport
```

Comment:
vyper question, is there any gas reduction benefit of putting the currentStrategy params in a variable like
` strategy: StrategyParams = self.strategies[_strategy]` to be used in several lines or is this optimized by compiler already?

Reply:
It's actually worse, because it does `size(StrategyParams)` number of `SLOAD`s, vs just 1